daemon/policy.go

```diff
@@ -77,46 +76,65 @@ func (d *Daemon) TriggerPolicyUpdates(added []policy.NumericIdentity) *sync.Wait
 		// FIXME: Invalidate only cache that is affected
 		d.invalidateCache()
 	}
-
-	d.GetPolicyRepository().Mutex.RLock()
-	d.EnablePolicyEnforcement()
```

Comment:
It's out of place here given how this patch changes how we handle configuration updates. his is handled as part of the handler for configuration changes, which calls `d.EnablePolicyEnforcement` before it calls `TriggerPolicyUpdates`. Since `TriggerPolicyUpdates` is triggered when the daemon's configuration is changed for `PolicyEnforcement`. `EnablePolicyEnforcement()` doesn't actually change any configuration in the daemon any more in this patch, so running it has no effect here anyway.